gallery/src/jsUtilPages/VulnerabilityLevelUtils/VulnerabilityLevelUtilsPage.tsx

```diff
@@ -0,0 +1,79 @@
+
+const VulnerabilityLevelUtilsPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        The React Shared Components library includes several simple JavaScript
+        objects and TypeScript types that provide code definitions and mappings to help represent
+        CVSS scores and severity levels.
+      </NxP>
+      <NxList bulleted>
+        <NxList.Item>
+          <NxList.Text>
+            <strong>Vulnerability (CVSS) Score:</strong>{' '}
+            A number with one decimal point precision between 0.0 and 10.0, inclusive,
+            that describes the severity of a threat. Zero represents no threat at all,
+            and ten represents the most severe of threats.
+          </NxList.Text>
+        </NxList.Item>
+        <NxList.Item>
+          <NxList.Text>
+            <strong>Vulnerability Severity Level</strong>{' '}
+            In increasing order of severity: <q>none</q>; <q>low</q>; <q>medium</q>; <q>high</q>;
+            and <q>critical</q>. Each vulnerability severity level is a member of a corresponding
+            range of CVSS Scores:
+
+            <NxList className="nx-list--bulleted">
+              <NxList.Item>
+                <NxList.Text><strong>none:</strong> 0.0</NxList.Text>
+              </NxList.Item>
+              <NxList.Item>
+                <NxList.Text><strong>low:</strong> 0.1{'\u2014'}3.9</NxList.Text>
+              </NxList.Item>
+              <NxList.Item>
+                <NxList.Text><strong>medium:</strong> 4.0{'\u2014'}6.9</NxList.Text>
+              </NxList.Item>
+              <NxList.Item>
+                <NxList.Text><strong>high:</strong> 7.0{'\u2014'}8.9</NxList.Text>
+              </NxList.Item>
+              <NxList.Item>
+                <NxList.Text><strong>critical:</strong> 9.0{'\u2014'}10.0</NxList.Text>
+              </NxList.Item>
+            </NxList>
+          </NxList.Text>
+        </NxList.Item>
+      </NxList>
+    </GalleryDescriptionTile>
+
+    <GalleryExampleTile title="formatVulnerabilityScore"
+                        codeExamples={formatVulnerabilityScoreExampleCode}>
+      <NxCode>formatVulnerabilityScore</NxCode>{' '}
+      is a helper function that formats a number
+      into a valid Vulnerability (CVSS) score string representation.
+      This ensures the score is valid and displayed consistently.
+      This function will throw an error if the number is not within the valid range (0.0 to 10.0),
+      or if the number is not within one decimal place.
+    </GalleryExampleTile>
+
+    <GalleryExampleTile title="getVulnerabilitySeverityLevel"
+                        codeExamples={getVulnerabilitySeverityLevelExampleCode}>
+      <NxCode>getVulnerabilitySeverityLevel</NxCode>{' '}
+      is a helper function that categorizes a CVSS score into its corresponding severity level.
+      This ensures that the severity level is accurately determined based on the CVSS score.
+    </GalleryExampleTile>
```

Comment:
There's only two things documented here but 7 things actually exported from the file.

Reply:
> There's only two things documented here but 7 things actually exported from the file.

Added examples here: [d726e8b](https://github.com/sonatype/sonatype-react-shared-components/pull/1202/commits/d726e8b0276c802ab759560cefb7dadee6daeca2)